port/unix/omrsysinfo.c

```diff
@@ -7610,7 +7610,102 @@ read_fully(struct OMRPortLibrary *portLibrary, intptr_t file, char **data, uintp
 uintptr_t
 omrsysinfo_get_processes(struct OMRPortLibrary *portLibrary, OMRProcessInfoCallback callback, void *userData)
 {
-#if defined(LINUX)
+#if defined(AIXPPC)
+	pid_t procIndex = 0;
+	int32_t maxProcs = 256;
```

Comment:
The types of locals should match their uses in connection with system functions: 

`maxProcs`, `argsSize`, `i`, `numProcs`, `idx`, `newSize`, `scanLimit` should all be of type `int`.